### ISSUE TITLE
Missing chart of active repositories over time

### ISSUE THREAD

**Issue by jsmanrique** [chaoss/grimoirelab-sigils#272]
Wouldn't it be nice having a chart showing the evolution of active repositories (git, GitHub, GitLab, etc) over time? At least it would be useful in the _overview_ panels

**Comment by alpgarcia** [comment]
What does it mean `evolution of active repositories`? Are we talking about the number?

After clarifying that, I would say we need to build a new panel instead of adding more things to the current one, so would you have in mind a use case in which this chart could be useful?

**Comment by jsmanrique** [comment]
I think that a panel related with _activity_ that shows the evolution of active repositories might be useful. Because currently the panels only show the total number of repositories active for a time frame.

What I have on mind is a Timelion chart for example, like this one for Git repositories:
```
.es(index=git, timefield=grimoire_creation_date, metric=cardinality:github_repo).label('Active git repositories'), .es(index=git, timefield=grimoire_creation_date, metric=cardinality:github_repo).trend().label('Active git repos trend')

```

**Comment by alpgarcia** [comment]
I guess this was already solved via #309

https://chaoss.github.io/grimoirelab-sigils/panels/lifecycle/
